### ISSUE TITLE
Component construction fails if CSS field doesn't contain curly brackets

### ISSUE THREAD

**Issue by trvrm** [ractivejs/ractive#3005]
## Description:
Component construction fails if CSS field doesn't contain curly brackets
## Versions affected:

0.9.  

This is a regression: 0.8 worked fine.

## Reproduction:

Full demonstration here: https://codepen.io/trvrm/pen/dRpjoe

The CSS field has no curly brackets in it,  so Ractive treats it as a selector, causing the component to never get constructed.

The same happens if this CSS is just a non-zero number of whitespace characters.

`hasCurly` (fairly reasonably) doesn't think this string is valid CSS

https://github.com/ractivejs/ractive/blob/63d8eef9fbff67e1a059c7089adc6db7c616a12d/src/Ractive/config/custom/css/css.js#L7

## Code:

```coffeescript

Ractive.components.Component1 = Ractive.extend
    # Remove/rename the following line and the component will get constructed
    css:' /*comment*/'
    template: 'the template'
```

**Comment by ghost** [comment]
This was probably done as a quick, non-comprehensive check of CSS vs selector. To be fair, the string is neither CSS nor selector and an error either way is correct. 😄

**Comment by evs-chris** [comment]
This should now be fixed in edge and will go out with 0.9.2 when it's cut. Thanks for the report!
